class.csstidy_print.php

```diff
@@ -350,7 +350,9 @@ public function _convert_raw_css($default_media='') {
 
 		// important comment section ?
 		if (isset($this->css['!'])) {
-			$this->parser->_add_token(IMPORTANT_COMMENT, rtrim($this->css['!']), true);
+			if (is_string($this->css['!'])) {
+				$this->parser->_add_token(IMPORTANT_COMMENT, rtrim($this->css['!']), true);
+			}
```

Comment:
I will not add this patch without a use cas to reproduce it, as I am not sure if the correct fix is this or a cast on `$this->css['!']` (ie should the `_add_token(IMPORTANT_COMMENT...)` append or not?)

Reply:
Hi @Cerdic and thank you for reviewing!

We received an email with the following style block that caused an error:
```
<style type="text/css">
            table, td { font-family: \'Noto Sans\', Helvetica, Arial, sans-serif; }
        /*!* CLIENT-SPECIFIC STYLES *!*/
        body, table, td, a { -webkit-text-size-adjust: 100%; -ms-text-size-adjust: 100%;}
        table, td { mso-table-lspace: 0pt; mso-table-rspace: 0pt; }
        img { -ms-interpolation-mode: bicubic; }

        /*!* RESET STYLES *!*/
        img { border: 0; line-height: 100%; outline: none; text-decoration: none; }
        table { border-collapse: collapse !important; }
        body { height: 100% !important; Margin: 0 !important; padding: 0 !important; width: 100% !important; }
...
```

I am not sure this is the correct fix, I only made this as a quick fix solution to stop errors in our project.

Thanks again